lib/datasets.go

```diff
@@ -582,13 +585,13 @@ type RenderParams struct {
 	Ref string `json:"ref"`
 	// Optionally pass an entire dataset in for rendering, if providing a dataset,
 	// the Ref field must be empty
-	Dataset *dataset.Dataset
+	Dataset *dataset.Dataset `json:"dataset"`
 	// Optional template override
-	Template []byte
+	Template []byte `json:"template"`
 	// If true,
```

Comment:
Just noticed this incomplete comment. I think `UseFSI` was removed everywhere, though I'm not 100% sure if render needs it. Maybe just add a TODO to investigate.